codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/PresignerGenerator.kt

```diff
@@ -105,6 +119,40 @@ class PresignerGenerator : SwiftIntegration {
         }
     }
 
+    private fun renderPresignAPIInServiceClient(
+        writer: SwiftWriter,
+        clientName: String,
+        op: OperationShape,
+        inputType: String
+    ) {
+        writer.apply {
+            openBlock("extension $clientName {", "}") {
+                val params = listOf("input: $inputType", "expiration: Foundation.TimeInterval")
+                val returnType = "URLRequest"
+                renderDocForPresignAPI(this, op, inputType)
+                openBlock("public func presignRequestFor${op.toUpperCamelCase()}(${params.joinToString()}) async throws -> $returnType {", "}") {
```

Comment:
`public func presignedRequestFor...` (see comment on URL presign renderer.)

Reply:
Changed as suggested.

Reply:
I don't see this change in the latest diff.  Maybe check to see if you've pushed all to github?